include/SDL3/SDL_video.h

```diff
@@ -2854,6 +2868,19 @@ extern SDL_DECLSPEC bool SDLCALL SDL_SetWindowProgressState(SDL_Window *window,
  */
 extern SDL_DECLSPEC bool SDLCALL SDL_SetWindowProgressValue(SDL_Window *window, float value);
 
+/**
+* Get the value of the progress bar for the given window’s taskbar icon.
+*
+* \param window the window to get the current progress value from.
+* \returns the progress value, (0.0f - start, 1.0f - end), or -1.0f on
```

Comment:
```suggestion
* \returns the progress value in the range of [0.0 - 1.0], or -1 on
```
I'm on the fence about whether this should return an error value or just 0.0 for no progress. Thoughts?

Reply:
For this one, I shamelessly copied the SDL_GetWindowOpacity function, which states:
``` C
 * \returns the opacity, (0.0f - transparent, 1.0f - opaque), or -1.0f on
 *          failure; call SDL_GetError() for more information.
```
For consistency, it may be better to return -1.0f on error. Besides, the SDL_GetWindowProgressValue Win32 implementation can't really fail except if it is unsupported, because this system doesn't provide any API for this and we have to handle it ourselves by simply returning the manually stored value. In this situation, returning 0 could be better. But on other platforms, this getter may have to call system functions which may lead to some errors, in which case I think having a distinct error value would be more appropriate.

Reply:
Oh, I interpreted the `- start` as a "minus start value", lol. I think your change is fine and we can improve the SDL_GetWindowOpacity() documentation later.